puzzle.css

```diff
@@ -21,11 +21,14 @@
     font-weight: bold;
     font-family: monospace;
     border: none;
-    color: transparent;
-    text-shadow: 0 0 0 black, -2px -2px 0 #fff, 0 -2px 0 #fff, 2px -2px 0 #fff, 2px 0 0 #fff, 2px 2px 0 #fff, 0 2px 0 #fff, -2px 2px 0 #fff, -2px 0 0 #fff;
+    color: black;
+    text-shadow: -2px -2px 0 #fff, 0 -2px 0 #fff, 2px -2px 0 #fff, 2px 0 0 #fff, 2px 2px 0 #fff, 0 2px 0 #fff, -2px 2px 0 #fff, -2px 0 0 #fff;
+    user-select: none;
+    cursor: pointer;
+    text-transform: uppercase;
 }
 
```

Comment:
I found the focus border hard to see. Can we add

```
.puzzle-entry td:focus .text {
    box-shadow: 0px 0px 3px 3px Highlight;
    box-shadow: 0px 0px 3px 3px -webkit-focus-ring-color;
}
```

Reply:
Hmm, once this change is in, the above change is much easier to see :-)

Reply:
I'm sorry, I'm not quite sure what you mean?

Reply:
the tabindex=0 didn't seem to do anything but now it does